src/backend/marsha/core/tests/api/playlist_accesses/test_create.py

```diff
@@ -0,0 +1,212 @@
+"""Tests for the PlaylistAccess create API of the Marsha project."""
+
+from django.test import TestCase
+
+from marsha.core import factories, models
+from marsha.core.simple_jwt.factories import UserAccessTokenFactory
+
+
+class PlaylistAccessCreateAPITest(TestCase):
+    """Test the create API for playlist access objects."""
+
+    maxDiff = None
+
+    def assertPostReturns403(self, playlist, user):
+        """Assert that a POST request returns a 403 response."""
```

Comment:
```suggestion
    def assert_user_cant_create_playlist_access(self, playlist, user):
        """Assert that a user can't create a playlist access with a POST request."""
```
WDYT ?

Reply:
I was wondering what is the better, I follow your opinion.